apps/mobile/eas.json

```diff
@@ -20,9 +20,11 @@
     "maestro": {
       "extends": "base",
       "environment": "production",
-      "withoutCredentials": true,
       "channel": "maestro",
-      "distribution": "internal"
+      "distribution": "internal",
+      "ios": {
```

Comment:
I need to add `simulator: true` here or else the simulator won't run in EAS 

![Screenshot 2025-06-25 at 12 05 58](https://github.com/user-attachments/assets/547f0b46-de7d-4905-b6d7-eb51491dac7b)